artifactory/commands/mvn/mvn.go

```diff
@@ -21,6 +21,7 @@ import (
 )
 
 const mavenExtractorDependencyVersion = "2.25.0"
+const mavenExtractorDependencyJCenterVersion = "2.23.0"
```

Comment:
Let's a descreiption for this constant. It is best to include the word "deprecated" to the description, so that we remember to remove it in jfrog-cli 2.0.
The same applies for the ```gradleExtractorDependencyJCenterVersion``` const.